src/content/updates/migration/developers.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [Repo architecture](#repo-architecture)
+- [Migration tooling](#migration-tooling)
+- [Migration strategy](#migration-strategy)
+- [Carbon components](#carbon-components)
+- [Carbon elements](#carbon-elements)
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Repo architecture
+
+Two of Carbon's previous repos will be deprecated with v10: carbon-icons and carbon-themes. Iconography and themes in v10 are now handled as packages in the carbon-elements repo. 
+
+[@carbon/colors](https://github.com/IBM/carbon-elements/tree/master/packages/colors)
+
+[@carbon/themes](https://github.com/IBM/carbon-elements/tree/master/packages/themes)
+
+## Migration tooling
+
+**Coming soon!** [carbon-upgrade](https://github.com/carbon-design-system/carbon-upgrade/blob/master/README.md) is a new tool to help developers upgrade to v10 through the command line. This tool is still experimental, and if you would like to contribute, the Carbon team would welcome the help! [Please leave ideas for migration tooling here](https://github.com/carbon-design-system/carbon-upgrade/issues/7). 
+
+## Migration strategy
+
+It is up to each product team to determine which migration strategy works best for their product. Because each team is different, there is no "official playbook" for migrating to v10. However, the Carbon team recommends the following best practices for migration.
+
+### Talk about migration early and often. 
+Meet with all relevant stakeholders from design, OM, and engineering to kick off the migration process. Discuss how the migration process can integrate into the team's regular cadence. Establish dedicated channels of communication for migration, and involve the broader team as needed.
+
+### Migrate in a dedicated branch.
+Rather than trying to upgrade piece-by-piece on the production build, the Carbon team recommends that product teams perform the migration first in a separate branch (or branches). This allows design and dev to work together and iron out any rough spots that may appear in the UI after upgrading. Don't push to production until the product matches the look and feel of the new IBM Design Language.
```

Comment:
"Don't merge into your primary development branch until..." instead of "Don't push to production until..." because completing migration does not mean deployment.

Reply:
Fixed.